Add api method createcryptopaymentacct

ghubstan opened the pull request:
This change supports creation of `BSQ` `BLOCKCHAIN` payment method accounts.

- Added proto message defs to `grpc.proto`.
- Added grpc server boilerplate to `GrpcPaymentAccountsService`.
- Added server impl to `CoreApi`, `CorePaymentAccountsService`.
- Added `createcryptopaymentacct-help.txt`.
- Added CLI side support for new api method.
- Added opt parsing unit tests to `OptionParsersTest`.

This is the 1st PR in a series, with the goal of supporting the BTC/BSQ trading pair.  Support for other crypto currency payment accounts will be added later.

ghubstan committed:
Add api method createcryptopaymentacct

This change supports creation of BSQ BLOCKCHAIN payment method accounts.

- Added proto message defs to grpc.proto.
- Added grpc server boilerplate to GrpcPaymentAccountsService.
- Added server impl to CoreApi, CorePaymentAccountsService.
- Added createcryptopaymentacct-help.txt.
- Added CLI side support for new api method.
- Added opt parsing unit tests to OptionParsersTest.

This is the 1st PR in a series, with the goal of supporting the BTC/BSQ trading
pair.  Support for other crypto currency payment accounts will be added later.

ghubstan committed:
Add cli side help for createcryptopaymentacct

wallclockbuilder reviewed (dismissed):
ACK

`./bisq-cli --password=xyz --port=9998 createcryptopaymentacct \
    --account-name="My BSQ Account" \
    --currency-code=BSQ \
    --address=BmxdvDCer85sKFWGc5wWTk2JcmK13jWGZsJ`

payment account saved
Name                     Currency    Payment Method  ID                                                    
My BSQ Account  BSQ            BLOCK_CHAINS    4222730d-802d-4c23-87ed-db096e6a696f 

`./bisq-cli --password=xyz --port=9998 createcryptopaymentacct \
    --account-name="My BSQ Account" \
    --currency-code=BSQ \
    --address=wrongaddressr85sKFWGc5wWTk2JcmK13wrongaddress`

Error: wrongaddressr85sKFWGc5wWTk2JcmK13wrongaddress is not a valid bsq address

ghubstan committed:
Merge branch 'master' into 01-api-add-createcryptopaymentacct

ghubstan committed:
Add support for creating instant altcoin payment accounts in api

- Added bool tradeInstant field to proto message def.
- Adjusted core createcryptopaymentacct impl to new tradeInstant request param.
- Adjusted cli side createcryptopaymentacct impl to new tradeInstant request param.
- Fixed CliMain's takeoffer help text (was missing the --payment-account opt).

ghubstan committed:
Remove space char
